Dockerfile

```diff
@@ -2,14 +2,18 @@
 #
 # Use of this source code is governed by the LICENSE file in this repository.
 
+# set a global Docker argument for the default CLI version
+#
+# https://github.com/moby/moby/issues/37345
+ARG TERRAFORM_VERSION=0.12.26
 
 ################################################################################
 ##     docker build --no-cache --target binary -t vela-terraform:binary .     ##
 ################################################################################
 
 FROM alpine:latest as binary
 
-ENV TF_VERSION=0.12.24
+ARG TF_VERSION
```

Comment:
do we want to set this to `TERRAFORM_VERSION`?

Reply:
Yes 😞